CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ## [Unreleased]
 
+## [2.1.1] - 2019-03-04
```

Comment:
Can you please move your changelog entry to Unreleased.  We will move it to the released version when a new release is created.

Reply:
Also done 👍